tests/autoyast/installation.pm

```diff
@@ -289,9 +289,10 @@ sub run {
         }
         elsif (match_has_tag('nvidia-validation-failed')) {
             # nvidia repositories are unstable and really not needed for anything
-            record_info("nVidia", "nVidia repository is broken");
+            record_info("NVIDIA", "NVIDIA repository is broken");
             send_key 'alt-y';
```

Comment:
there is only OK button, I don't think we need here to press 'Yes'.

Reply:
I'm not sure why it press 'Yes' at here. Maybe other case need this one. It was written by others.

Reply:
but the needle didn't match in OSD so we are supposed to do whatever we want here, nobody is using it (I hope :) )
`alt-y` probably was there because there was a different dialog with a `yes`.